assets/src/blocks/CarouselHeader/CarouselHeaderEditor.js

```diff
@@ -11,129 +11,127 @@ import { Sidebar } from './Sidebar';
     ];
     const lastSlide = newSlides.length - 1;
-    setAttributes({ slides: newSlides });
-    goToSlide(currentSlide > lastSlide ? 0 : currentSlide, true);
-  }
+    setAttributes( { slides: newSlides } );
+    goToSlide( currentSlide > lastSlide ? 0 : currentSlide, true );
+  };
 
-  const needsMigration = slides.some(slide => !!slide.image && !slide.image_srcset)
-  const migratedSlides = useSelect(select=> slides && slides.map(slide => {
-    if (!needsMigration) {
-      return;
+  const needsMigration = slides.some( ( slide ) => !! slide.image && ! slide.image_srcset );
+  const migratedSlides = useSelect( ( select ) => slides && slides.map( ( slide ) => {
+    if ( ! needsMigration ) {
+      return null;
     }
     let attempt = 0;
     let image;
     // Run a loop as this could return undefined the first time.
-    while (!image && attempt++ < 100) {
-      image = select('core').getMedia(slide.image);
+    while ( ! image && attempt++ < 100 ) {
+      image = select( 'core' ).getMedia( slide.image );
     }
     // Didn't see this case occur but catch it anyway.
-    if (!image) {
-      console.log('Failed fetching');
+    if ( ! image ) {
       return slide;
     }
-    const image_srcset = toSrcSet(Object.values(image.media_details.sizes))
-    return ({ ...slide, image_url: image.source_url, image_srcset, image_alt: image.alt_text });
-  }), [needsMigration]);
+    const image_srcset = toSrcSet( Object.values( image.media_details.sizes ) );
+    return ( { ...slide, image_url: image.source_url, image_srcset, image_alt: image.alt_text } );
+  } ), [ needsMigration ] );
 
   return (
-    <section className={`block block-header alignfull carousel-header ${className ?? ''}`}>
+    <section className={ `block block-header alignfull carousel-header ${ className ?? '' }` }>
       <Sidebar
-        carouselAutoplay={carousel_autoplay}
-        slides={slides}
-        setAttributes={setAttributes}
-        currentSlide={currentSlide}
-        changeSlideAttribute={changeSlideAttribute}
-        goToSlide={goToSlide}
+        carouselAutoplay={ carousel_autoplay }
+        slides={ slides }
+        setAttributes={ setAttributes }
+        currentSlide={ currentSlide }
+        changeSlideAttribute={ changeSlideAttribute }
+        goToSlide={ goToSlide }
       />
       { needsMigration && <button
-        title={'This block was created before WYSIWYG, press this to fetch data the new version uses.'}
+        title={ 'This block was created before WYSIWYG, press this to fetch data the new version uses.' }
         onClick={ () => {
-          setAttributes({ slides: migratedSlides });
+          setAttributes( { slides: migratedSlides } );
         } }
-      >Migrate image data</button>}
-      <div className='carousel-wrapper-header'>
-        <div className='carousel-inner' role='listbox'>
-          {slides?.map((slide, index) => (
+      >Migrate image data</button> }
+      <div className="carousel-wrapper-header">
```

Comment:
I'm not sure why in some places single quotes are being replaced to double ones. 
Which is the rule we should follow?